arrayfire_test compilation error

Hello
This is 'just' a compilation error with todays master when building AF in a build folder not called 'build' (eg 'release', or 'debug'):
...
[ 23%] Built target af_cuda_static_cuda_library
In file included from /home/wtambellini/repos/arrayfire/test/./testHelpers.hpp:17:0,
                 from /home/wtambellini/repos/arrayfire/test/arrayfire_test.cpp:11:
/home/wtambellini/repos/arrayfire/test/../include/af/array.h:11:26: fatal error: af/compilers.h: No such file or directory
 #include <af/compilers.h>
                          ^
compilation terminated.

This is probably because the test cmake hardcodes the build folder to be named 'build' :  

target_include_directories(arrayfire_test
  PRIVATE
    .
    ../include
    ../build/include
    ../extern/half/include
    mmio
    gtest/googletest/include)

--- a/test/CMakeLists.txt
+++ b/test/CMakeLists.txt
@@ -78,10 +78,10 @@ add_library(arrayfire_test OBJECT
 
 target_include_directories(arrayfire_test
   PRIVATE
-    .
-    ../include
-    ../build/include
-    ../extern/half/include
+    ${CMAKE_CURRENT_LIST_DIR}
+    ${ArrayFire_SOURCE_DIR}/include
+    ${ArrayFire_BINARY_DIR}/include
+    ${ArrayFire_SOURCE_DIR}/extern/half/include
     mmio
     gtest/googletest/include)
 

